install_config/install/advanced_install.adoc

```diff
@@ -2365,6 +2380,74 @@ endif::[]
 documentation assume use of the RPM-based installer, so use this relative path
 instead when using the containerized installer.
 
+[[running-the-advanced-installation-individual-components]]
+=== Running Individual Component Playbooks
+
+The main installation playbook *_{pb-prefix}playbooks/byo/config.yml_* runs a
+set of individual component playbooks in a specific order, and the installer
+reports back at the end what phases you have gone through. If the installation
+fails during a phase, you are notified on the screen along with the errors from
+the Ansible run.
+
+After you resolve the issue, rather than run the entire installation over again,
+you can pick up from the failed phase. You must then run each of the remaining
+playbooks in order:
+
+----
+# ansible-playbook  [-i /path/to/inventory] <playbook_file_location>
+----
+
+The following table is sorted in order of when each individual component
+playbook is run:
+
+.Individual Component Playbook Run Order
+[options="header",cols="1,3"]
+|===
+|Playbook Name |File Location
+
+|Health Check
+|*_{pb-prefix}playbooks/byo/openshift-checks/pre-install.yml_*
+
+|etcd Install
+|*_{pb-prefix}playbooks/byo/openshift-etcd/config.yml_*
+
+|NFS Install
+|*_{pb-prefix}playbooks/byo/openshift-nfs/config.yml_*
+
+|Load Balancer Install
+|*_{pb-prefix}playbooks/byo/openshift-loadbalancer/config.yml_*
+
+|Master Install
+|*_{pb-prefix}playbooks/byo/openshift-master/config.yml_*
+
+|Master Additional Install
+|*_{pb-prefix}playbooks/byo/openshift-master/additional_config.yml_*
+
+|Node Install
+|*_{pb-prefix}playbooks/byo/openshift-node/config.yml_*
+
+|GlusterFS Install
+|*_{pb-prefix}playbooks/byo/openshift-glusterfs/config.yml_*
+
+|Hosted Install
+|*_{pb-prefix}playbooks/byo/openshift-cluster/openshift-hosted.yml_*
+
+|Metrics Install
+|*_{pb-prefix}playbooks/byo/openshift-cluster/openshift-metrics.yml_*
+
+|Logging Install
+|*_{pb-prefix}playbooks/byo/openshift-cluster/openshift-logging.yml_*
+
+|Prometheus Install
```

Comment:
According to playbook order listed in playbooks/common/openshift-cluster/config.yml, seem like there is no single entrypoint to call openshift-prometheus.yml, while it is already included in openshift_hosted.yml, so I think we need remove this line here.

Reply:
Prometheus does have an entry point [here](https://github.com/openshift/openshift-ansible/blob/release-3.7/playbooks/byo/openshift-cluster/openshift-prometheus.yml.  The call to common/openshift-cluster/openshift_prometheus.yml should not be in common/openshift-cluster/openshift_hosted.yml, but should rather be in common/openshift-cluster/config.yml.  I noticed this yesterday when doing the refactoring for 3.8.  I will put in a PR to update the 3.7 branch accordingly.

Reply:
Yeah, totally agree to move the call for openshift-prometheus.yml from common/openshift-cluster/openshift_hosted.yml to common/openshift-cluster/config.yml. If that, agree to keep this line here.